src/index.ts

```diff
@@ -56,6 +56,15 @@ interface Colors {
   backgroundColor?: string;
 }
 
+interface Styles extends Colors {
```

Comment:
Hmmm. Wonder if at this point it makes sense to take the styles attribute as a `CSSProperties` type and just allow people to provide a set of attributes directly. Might be more maintainable then building up our own set of mappings.

Reply:
@captainsafia we would like to use your module directly, we had our own solution with anser before. 

the styles provided by @vmik7 are the ones supported by anser: https://github.com/IonicaBizau/anser/blob/ac20b53394a933ac9a2364159f49dd6903f08682/lib/index.js#L598-L610

i agree with the interface, do you want us to send a PR building on top of @vmik7's commit?

Reply:
i felt very ambitious @captainsafia, here is the PR: https://github.com/nteract/ansi-to-react/pull/74

it keeps the commit from @vmik7 who did the main work :)